packages/data-service/src/instance-detail-helper.ts

```diff
@@ -133,12 +135,19 @@ export async function getInstance(
       getParameter: 1,
       featureCompatibilityVersion: 1,
     }).catch(() => null),
-
-    runCommand(adminDb, { atlasVersion: 1 }).catch(() => {
-      return { atlasVersion: '', gitVersion: '' };
-    }),
+    checkIsLocalAtlas(
+      async (db: string, collection: string, query: Document) => {
+        const res = await runCommand(client.db(db), {
+          count: collection,
+          query,
+        });
+        return res.n;
```

Comment:
Similar to my [comment in the mongosh PR for this](https://github.com/mongodb-js/mongosh/pull/1629#discussion_r1309091281), I don’t think we’ll want to be using the `count` command and instead use the driver wrapper, if possible?